frontend/packages/core/src/tab.tsx

```diff
@@ -81,12 +81,18 @@ const TabPanel = styled(MuiTabPanel)({
 
 export interface TabsProps extends Pick<MuiTabsProps, "value" | "variant"> {
   children: React.ReactElement<TabProps> | React.ReactElement<TabProps>[];
+  // To allow for callback functionality, use `onChange`
+  // Note the value is referring to the index, like "0", "1", "2", etc.
+  onChange?: (value: string) => void;
```

Comment:
Actually I'm a bit confused as to why this is a string to begin with. The [docs](https://mui.com/material-ui/api/tabs/) are a bit unclear since it's an any. Should this be an int?
```
value: We default to the index of the child (number)
```

Reply:
I think the docs are wrong. If you set it as an any and log the type when it executes it comes out as a string

Reply:
<img width="991" alt="Screen Shot 2022-05-03 at 2 53 26 PM" src="https://user-images.githubusercontent.com/66325812/166572586-cbb06149-372f-4442-9714-5ade048ead9c.png">
In other places though it says its a string